test-containers/bnd-tools-smoke-test/app/src/test/java/com/vaadin/flow/osgi/smoketest/WaitHttpIT.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
```

Comment:
Tests can be without header or at least use the same cvdl header as the sources.

Reply:
Done